### PR TITLE
re-add icp tokens page

### PR DESCRIPTION
**Pull request by reigj1**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: src/pages/icp-tokens.tsx [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+                participate in decentralisation swaps to become a co-owner of an
+                SNS DAO, and when using many smart contract services built on
+                the Internet Computer such as registries, marketplaces and
+                exchanges.
+              </motion.p>
+              <motion.p
+                className="tw-paragraph md:tw-lead-sm text-black/60 mb-0"
+                variants={transitions.item}
+              >
+                The ICP token implements the ICRC-1 standard.
```
Comment (jessiemongeon1) [inline src/pages/icp-tokens.tsx:162]: ICP is *compatible* with the ICRC-1 standard, but it uses a different account format and stores block data differently: https://internetcomputer.org/docs/current/developer-docs/defi/tokens/ledger/overview#icp-ledger

Comment (jessiemongeon1) [inline src/pages/icp-tokens.tsx:162]:
```suggestion
                The ICP token is compatible with the ICRC-1 standard.
```

File: src/pages/icp-tokens.tsx [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+              </motion.figure>
+              <motion.figure
+                className="flex flex-col items-center m-0"
+                variants={transitions.item}
+              >
+                <span className="tw-heading-3 lg:tw-heading-60 text-gradient mb-2">
+                  ${(cryptoPrice.icp * 0.0001).toFixed(5)}
+                </span>
+                <figcaption className="tw-paragraph md:tw-lead-sm">
+                  Ledger TX Fee
```
Comment (jessiemongeon1) [inline src/pages/icp-tokens.tsx:246]:
```suggestion
                  Ledger TX fee
```

File: src/pages/icp-tokens.tsx [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+                <p className="tw-paragraph-sm mb-3 text-black/60">
+                  Community discussions around extending the ICRC token standard
+                  for all ledgers to optimize functionality is ongoing.
+                </p>
+                <p className="mb-0">
+                  <Link
+                    href="https://forum.dfinity.org"
+                    className="link-primary link-with-icon"
+                  >
+                    Share your ideas on the Forum
```
Comment (jessiemongeon1) [inline src/pages/icp-tokens.tsx:744]:
```suggestion
                    Share your ideas on the forum
```

File: src/pages/icp-tokens.tsx (hunk 1 above)
Comment (reigj1) [reply src/pages/icp-tokens.tsx]: Thanks @jessiemongeon1, feel free to update directly here on this PR

**Review by meodai — APPROVED** [review]

**Pull request merged**